spec/assisted_workflow/addons/git_spec.rb

```diff
@@ -5,18 +5,18 @@
   before do
     @git = AssistedWorkflow::Addons::Git.new(nil, :silent => true)
     stub(@git).system_error?{ false }
-    stub(@git).system("git rev-parse --abbrev-ref HEAD"){ "flavio.00001.new_feature"}
+    stub(@git).system("git rev-parse --abbrev-ref HEAD"){ "flavio.1234.new_feature"}
```

Comment:
Line is too long. [84/80]<br>Space missing to the left of {.<br>Space missing inside }.